src/irmin-pack/unix/gc.ml

```diff
@@ -184,6 +184,24 @@ module Make (Args : Args) : S with module Args := Args = struct
     Bytes.set buffer Hash.hash_size magic_parent;
     write_exn ~off:poff ~len (Bytes.unsafe_to_string buffer)
 
+  let create_new_suffix ~root ~generation =
+    let open Result_syntax in
+    let path = Irmin_pack.Layout.V3.suffix ~root ~generation in
+    (* As the new suffix is necessarily in V3, the dead_header_size is 0. *)
```

Comment:
what's the [dead_header_size] ?

Reply:
I forgot to remove the comment, its not pertinent here -  its a field necessary when _opening_ append-only files, to be able to read files in version 1, 2 (that contain a header) and files in v3 (that do not contain a header but instead use the control file).